src/components/navigation-drawer/NavigationDrawer.js

```diff
@@ -29,7 +28,10 @@ export default {
     persistent: Boolean,
     right: Boolean,
     temporary: Boolean,
-    touchless: Boolean,
+    touchless: {
+      type: Boolean,
+      default: false
```

Comment:
This is redundant, Boolean type props default to undefined which evaluates to false.